test/jdk/java/lang/Double/ParseLargeStrings.java

```diff
@@ -19,30 +19,33 @@
  * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
  * or visit www.oracle.com if you need additional information or have any
  * questions.
- *
  */
 
-#ifndef SHARE_JFR_RECORDER_CHECKPOINT_TYPES_JFRTHREADGROUPMANAGER_HPP
-#define SHARE_JFR_RECORDER_CHECKPOINT_TYPES_JFRTHREADGROUPMANAGER_HPP
-
-#include "jfr/utilities/jfrTypes.hpp"
-#include "memory/allStatic.hpp"
-
-class JfrCheckpointWriter;
+/*
+ * @test
+ * @bug 8371652
+ * @summary Double.parseDouble with large strings should return Double Infinity
```

Comment:
Large strings can return _any_ `double`, not only infinity.
Think of 1 billions `' '` (or `'0'`) followed by a `'1'`, which returns `1.0`.
Or 1 billion `'9'` followed by `e-10000000000000000000`, which returns `0.0`.
And so on.